cmd/root.go

```diff
@@ -102,6 +107,34 @@ func logLevel() error {
 	return nil
 }
 
+func setupSentryHook() error {
+	sentryEnv := viper.GetString("sentry.env")
+	sentryDSN := viper.GetString("sentry.dsn")
```

Comment:
maybe it makes sense to return early if these aren't assigned? Have verified everything runs without them, but we do see stuff like this in the logs:
```
[Sentry] 2021/01/18 10:20:00 Sentry client initialized with an empty DSN. Using noopTransport. No events will be delivered.
[Sentry] 2021/01/18 10:20:00 Integration installed: ContextifyFrames
[Sentry] 2021/01/18 10:20:00 Integration installed: Environment
[Sentry] 2021/01/18 10:20:00 Integration installed: Modules
[Sentry] 2021/01/18 10:20:00 Integration installed: IgnoreErrors
```
I'll bet that would go away if we weren't initializing Sentry and adding a hook.

Reply:
Yeah, probably not huge overhead here but no harm in bailing early if the client's not using Sentry.